ui/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -154,13 +158,26 @@ const Layout: React.FC<RootComponentProps> = props => {
           <ScrollRestorer />
           <div className={tw(mobileLayoverStyle)}>
             <div className={tw(sidebarSlotStyle)}>
-              <div className={tw('pt-0 xl:pt-4 h-screen')} ref={sidebarWrapperRef}>
-                <Extension
-                  fullHeight
-                  name={props.layoutConfig.sidebarSlotId}
-                  uiEvents={props.uiEvents}
-                />
-              </div>
+              {needSidebarToggling ? (
+                <div className={tw('pt-0 xl:pt-4 h-screen')} ref={wrapperRef}>
```

Comment:
can you use Box component instead form DS core, then you may pass just plain style strings (without tw) to 'customStyle' prop